src/main/java/co/novu/common/base/Novu.java

```diff
@@ -454,7 +454,7 @@ public TopicResponse renameTopic(RenameTopicRequest request, String topicKey) th
         }
     }
 
-    public BulkIntegrationResponse getIntegrations() {
+    public BulkIntegrationResponse getIntegrations() throws Exception {
```

Comment:
Please specify the two exceptions being thrown
**_throws NovuNetworkException, IOException_**